localization/camera/CMakeLists.txt

```diff
@@ -54,38 +50,41 @@ catkin_package(
 # Specify additional locations of header files
 include_directories(
   include
-  ${catkin_INCLUDE_DIRS}
   ${EIGEN3_INCLUDE_DIR}
   ${OpenCV_INCLUDE_DIRS}
+  ${LUAJIT_INCLUDE_DIR}
 )
 
 # Declare C++ libraries
 add_library(camera
   src/camera_model.cc
   src/camera_params.cc
 )
-add_dependencies(camera ${catkin_EXPORTED_TARGETS})
-target_link_libraries(camera ${catkin_LIBRARIES} ${EIGEN_LIBRARIES} ${OpenCV_LIBRARIES})
+ament_target_dependencies(camera rclcpp ff_common config_reader)
+ament_export_targets(camera HAS_LIBRARY_TARGET)
+target_link_libraries(camera ${EIGEN_LIBRARIES} ${OpenCV_LIBRARIES})
 
-## Declare a C++ executable: undistort_image
+# Declare a C++ executable: undistort_image
 add_executable(undistort_image tools/undistort_image.cc)
-add_dependencies(undistort_image ${catkin_EXPORTED_TARGETS})
 target_link_libraries(undistort_image
-  camera gflags glog ${catkin_LIBRARIES})
+  camera gflags glog -lboost_filesystem -lboost_system)
```

Comment:
should use Boost_LIBRARIES

Reply:
interesting, I tried that first and didn't work...

Reply:
you also need to add filesystem to the boost components that's probably why

Reply:
worked like a charm!